### PR TITLE
Glossary entries for 'storage' and 'memory'

### PR DESCRIPTION
**Pull request by adamchalmers**

### CODE REVIEW

File: content/glossary/memory.mdx [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+synonyms:
+    - RAM
+domains:
+    - software-engineering
+---
+
+When you open a file or program, your computer loads it from storage into memory (also known as "random access memory" or RAM). Memory is the only place your computer can actually use and open files. Typical consumer laptops have between 8 and 16 gigabytes of memory, although top-end professional laptops might have 64. Every time you open a file or program, it takes up some memory. This means your computer's memory determines how many files and programs you can have open at once. If your computer is low on memory, it will slow down. You can close programs to free up memory.
+
+Memory is very different to storage. Files only consume memory when they're open, and when the file is closed that memory is freed up, ready for another program to use it. When you restart your computer, all programs are stopped and all files are closed. This means restarting the computer resets all the memory, which might make your computer faster when you turn it back on. Adding more memory to your computer won't let you store more files, but will let you open more files simultaneously.
```
Comment (adamchalmers) [inline content/glossary/memory.mdx:12]: @franknoirot Ideally I'd like "storage" here to hyperlink to the storage article, does our glossary support that?

Comment (franknoirot) [reply content/glossary/memory.mdx:12]: Yeah it does @adamchalmers! just use a relative markdown link, eg: `[storage](/docs/glossary/storage)`

I know it kinda sucks that the links aren't valid in any context other than the website docs, but I think that's okay for now.

Comment (franknoirot) [reply content/glossary/memory.mdx]: I went ahead and added 🤙🏻

**Review by franknoirot — APPROVED** [review]
Good clear writing, and all links work in preview ✨

**Pull request merged**